tests/provisioner.toml

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Static rust serialization for this file might look like

```rust
struct Config {
    packages: Option<Vec<String>>,
    groups: Option<Vec<Group>>,
    execs: Option<Vec<Exec>>, 
}
```
Or the programmer might skip `Option` for values they absolutely require.